config/compilerEnv.cmake

```diff
@@ -341,7 +343,12 @@ macro(dbsSetupStaticAnalyzers)
     # clang-tidy
     # https://clang.llvm.org/extra/clang-tidy/
     if( ${DRACO_STATIC_ANALYZER} STREQUAL "clang-tidy" )
-      find_program( CMAKE_CXX_CLANG_TIDY clang-tidy )
+      if( NOT CMAKE_CXX_CLANG_TIDY )
+        find_program( CMAKE_CXX_CLANG_TIDY clang-tidy )
+        get_filename_component(CT_BPATH ${CMAKE_CXX_CLANG_TIDY} DIRECTORY )
+        get_filename_component(CT_IPATH "${CT_BPATH}" DIRECTORY )
+        string(CONCAT CT_IPATH ${CT_IPATH} "/include/c++/v1")
+      endif()
       if( CMAKE_CXX_CLANG_TIDY )
         if( NOT "${CMAKE_CXX_CLANG_TIDY}" MATCHES "[-]checks[=]" )
           set( CMAKE_CXX_CLANG_TIDY "${CMAKE_CXX_CLANG_TIDY};-checks=mpi-*,bugprone-*,performance-*"
```

Comment:
Make the `-checks=` section set-able from the cmake command line so that nightly regressions or developers can easily select their own checks.